Subject: Escape hyphens in manpages
 Get rid of lintian minor warning "hyphen-used-as-minus-sign".
Author: Matteo Cypriani <mcy@lm7.fr>
Last-Update: 2021-01-30
--- a/doc/diskd.1
+++ b/doc/diskd.1
@@ -74,7 +74,7 @@
 .nf
 .ft 3
 .in +0.3i
-\&\fR\&\f(CWdiskd [\fR\&\f(CW-d \fIdrive\fR\&\f(CW] [\fR\&\f(CW-i \fIinterval\fR\&\f(CW] [\fR\&\f(CW-e \fIcommand\fR\&\f(CW]
+\&\fR\&\f(CWdiskd [\fR\&\f(CW\-d \fIdrive\fR\&\f(CW] [\fR\&\f(CW\-i \fIinterval\fR\&\f(CW] [\fR\&\f(CW\-e \fIcommand\fR\&\f(CW]
 .fi
 .in -0.3i
 .ft R
@@ -88,22 +88,22 @@
 .SH Warning
 .PP
 This program works by switching the motor on for a very short
-interval, and then seeking to track -1. This might damage hardware in
+interval, and then seeking to track \-1. This might damage hardware in
 the long run. Amigas, which also use these techniques, are known for
 having problems with their disk drives no longer spinning up properly
 after a few month of usage.
 .PP
 .SH Options
 .TP
-\&\fR\&\f(CW-d\ \fIdrive\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-d\ \fIdrive\fR\&\f(CW\fR\ 
 Selects the drive to observe for disk insertion. By default, drive 0
 (\fR\&\f(CW/dev/fd0\fR) is observed.
 .TP
-\&\fR\&\f(CW-i\ \fIinterval\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-i\ \fIinterval\fR\&\f(CW\fR\ 
 Selects the polling interval. The interval is given in tenths of
 seconds. Default is 10 (one second).
 .TP
-\&\fR\&\f(CW-e\ \fIcommand\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-e\ \fIcommand\fR\&\f(CW\fR\ 
 Gives the command to be executed when a disk is inserted. If no
 command is given the program simply exits. Typically, the command
 mounts the disk. It can be a shell scripts which probes for several
--- a/doc/fdmount.1
+++ b/doc/fdmount.1
@@ -72,19 +72,19 @@
 .nf
 .ft 3
 .in +0.3i
-\&\fR\&\f(CWfdmount [\fR\&\f(CW-l] [\fR\&\f(CW--list] [\fR\&\f(CW-d] [\fR\&\f(CW--daemon] [\fR\&\f(CW--detach]
-[\fR\&\f(CW-i \fIinterval\fR\&\f(CW] [\fR\&\f(CW--interval \fIinterval\fR\&\f(CW] [\fR\&\f(CW-o \fImount-options\fR\&\f(CW]
-[\fR\&\f(CW-r] [\fR\&\f(CW-readonly] [\fR\&\f(CW-s] [\fR\&\f(CW--sync] [\fR\&\f(CW--nosync] [\fR\&\f(CW--nodev]
-[\fR\&\f(CW--nosuid] [\fR\&\f(CW--noexec] [\fR\&\f(CW-f] [\fR\&\f(CW--force] [\fR\&\f(CW-h] [\fR\&\f(CW--help]
+\&\fR\&\f(CWfdmount [\fR\&\f(CW\-l] [\fR\&\f(CW\-\-list] [\fR\&\f(CW\-d] [\fR\&\f(CW\-\-daemon] [\fR\&\f(CW\-\-detach]
+[\fR\&\f(CW\-i \fIinterval\fR\&\f(CW] [\fR\&\f(CW\-\-interval \fIinterval\fR\&\f(CW] [\fR\&\f(CW\-o \fImount-options\fR\&\f(CW]
+[\fR\&\f(CW\-r] [\fR\&\f(CW\-readonly] [\fR\&\f(CW\-s] [\fR\&\f(CW\-\-sync] [\fR\&\f(CW\-\-nosync] [\fR\&\f(CW\-\-nodev]
+[\fR\&\f(CW\-\-nosuid] [\fR\&\f(CW\-\-noexec] [\fR\&\f(CW\-f] [\fR\&\f(CW\-\-force] [\fR\&\f(CW\-h] [\fR\&\f(CW\-\-help]
 [\fIdrivename\fR\&\f(CW] [\fImountpoint\fR\&\f(CW]
 \&\&
-\&\fR\&\f(CWfdumount [\fR\&\f(CW-f] [\fR\&\f(CW--force] [\fIdrivename\fR\&\f(CW]
+\&\fR\&\f(CWfdumount [\fR\&\f(CW\-f] [\fR\&\f(CW\-\-force] [\fIdrivename\fR\&\f(CW]
 \&\&
 \&\fR\&\f(CWfdlist
 \&\&
-\&\fR\&\f(CWfdmountd [\fR\&\f(CW-i \fIinterval\fR\&\f(CW] [\fR\&\f(CW--interval \fIinterval\fR\&\f(CW] [\fR\&\f(CW-r]
-[\fR\&\f(CW-readonly] [\fR\&\f(CW-s] [\fR\&\f(CW--sync] [\fR\&\f(CW--nosync] [\fR\&\f(CW--nodev]
-[\fR\&\f(CW--nosuid] [\fR\&\f(CW--noexec] [\fR\&\f(CW--help] [\fIdrivename\fR\&\f(CW] [\fImountpoint\fR\&\f(CW]]
+\&\fR\&\f(CWfdmountd [\fR\&\f(CW\-i \fIinterval\fR\&\f(CW] [\fR\&\f(CW\-\-interval \fIinterval\fR\&\f(CW] [\fR\&\f(CW\-r]
+[\fR\&\f(CW\-readonly] [\fR\&\f(CW\-s] [\fR\&\f(CW\-\-sync] [\fR\&\f(CW\-\-nosync] [\fR\&\f(CW\-\-nodev]
+[\fR\&\f(CW\-\-nosuid] [\fR\&\f(CW\-\-noexec] [\fR\&\f(CW\-\-help] [\fIdrivename\fR\&\f(CW] [\fImountpoint\fR\&\f(CW]]
 \&\&
 .fi
 .in -0.3i
@@ -112,42 +112,42 @@
 on \fR\&\f(CW\(if/fd[0-7]\(is\fR.  In either case, the mount point must be an
 existing, writable directory.
 .PP
-\&\fBDue to a bug in the floppy driver (?), the polling interval (-i
+\&\fBDue to a bug in the floppy driver (?), the polling interval (\-i
 flag) must be longer than the spindown offset. Thus you need to do (for
-example) floppycontrol --spindown 99 before starting fdmountd in daemon
+example) floppycontrol \-\-spindown 99 before starting fdmountd in daemon
 mode\fR
 .PP
 .SH Options
 .IP
 .TP
-\&\fR\&\f(CW-l\ \fI--list\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-l\ \fI\-\-list\fR\&\f(CW\fR\ 
 List all known drives with their symbolic name, type, and mount
 status. 
 .TP
-\&\fR\&\f(CW-d\ \fI--daemon\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-d\ \fI\-\-daemon\fR\&\f(CW\fR\ 
 Run in daemon mode (see below).
 .TP
-\&\fR\&\f(CW--detach\fR\ 
+\&\fR\&\f(CW\-\-detach\fR\ 
 Runs daemon in background, and detaches it from its tty. Messages
 produced after the fork are logged to syslog.
 .TP
-\&\fR\&\f(CW-p\ \fIfile\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-p\ \fIfile\fR\&\f(CW\fR\ 
 .TQ
-\&\fR\&\f(CW--pidfile\ \fIfile\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-pidfile\ \fIfile\fR\&\f(CW\fR
 .IP
 Dumps the process id of the daemon to
 \&\fIfile\fR. This makes killing the daemon easier:
-\&\fR\&\f(CWkill -9 `cat \fIfile\fR\&\f(CW`\fR
+\&\fR\&\f(CWkill \-9 `cat \fIfile\fR\&\f(CW`\fR
 .TP
-\&\fR\&\f(CW-i\ \fIinterval\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-i\ \fIinterval\fR\&\f(CW\fR\ 
 .TQ
-\&\fR\&\f(CW--interval\ \fIinterval\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-interval\ \fIinterval\fR\&\f(CW\fR
 Set the polling interval for daemon mode. The unit for \fIinterval\fR is
 0.1 seconds, the default value is 10 (i.e. 1 second).
 .TP
-\&\fR\&\f(CW-o\ \fIoptions\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-o\ \fIoptions\fR\&\f(CW\fR\ 
 .TQ
-\&\fR\&\f(CW--options\ \fIoptions\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-options\ \fIoptions\fR\&\f(CW\fR
 Sets filesystem-specific options. So far, these are only available for
 DOS and Ext2 disks. The following DOS options are supported:
 \&\fR\&\f(CWcheck\fR, \fR\&\f(CWconv\fR, \fR\&\f(CWdotsOK\fR, \fR\&\f(CWdebug\fR, \fR\&\f(CWfat\fR,
@@ -158,34 +158,34 @@
 options not applying to the disk that is inserted (because of its
 filesystem type) are not passed to mount.
 .TP
-\&\fR\&\f(CW-r\ \fI--readonly\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-r\ \fI\-\-readonly\fR\&\f(CW\fR\ 
 Mount the disk read-only. This is automatically assumed if the
 disk is write protected.
 .TP
-\&\fR\&\f(CW-s\ \fI--sync\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-s\ \fI\-\-sync\fR\&\f(CW\fR\ 
 Mount with the \fR\&\f(CWSYNC\fR option.
 .TP
-\&\fR\&\f(CW--nosync\fR\ 
+\&\fR\&\f(CW\-\-nosync\fR\ 
 Mounts without the \fR\&\f(CWSYNC\fR option, even when running as daemon.
 .TP
-\&\fR\&\f(CW--nodev\fR\ 
+\&\fR\&\f(CW\-\-nodev\fR\ 
 Mount with the \fR\&\f(CWNODEV\fR option. Ignored for \fR\&\f(CWmsdos\fR
 filesystems, otherwise always set for non-root users.
 .TP
-\&\fR\&\f(CW--nosuid\fR\ 
+\&\fR\&\f(CW\-\-nosuid\fR\ 
 Mount with the \fR\&\f(CWNOSUID\fR option. Ignored for \fR\&\f(CWmsdos\fR
 filesystems, otherwise always set for non-root users.
 .TP
-\&\fR\&\f(CW--noexec\fR\ 
+\&\fR\&\f(CW\-\-noexec\fR\ 
 Mount with the \fR\&\f(CWNOEXEC\fR option.
 .TP
-\&\fR\&\f(CW-f\ \fI--force\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-f\ \fI\-\-force\fR\&\f(CW\fR\ 
 Attempt a mount or unmount operation even \fR\&\f(CW\(if/etc/mtab\(is\fR says that
 the drive is already mounted, or not mounted, respectively.
 This option is useful if \fR\&\f(CW\(if/etc/mtab\(is\fR got out of sync with the
 actual state for some reason.
 .TP
-\&\fR\&\f(CW-h\ \fI--help\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-h\ \fI\-\-help\fR\&\f(CW\fR\ 
 Show short parameter description
 .PP
 .SH Security
@@ -224,7 +224,7 @@
 When the disk is removed, it is automatically unmounted.  However, it is
 recommended to unmount the disk manually \fIbefore\fR removing it. In
 order to limit corruption, disks are mounted with the SYNC option when
-running in daemon mode, unless the \fR\&\f(CW--nosync\fR flag is given.
+running in daemon mode, unless the \fR\&\f(CW\-\-nosync\fR flag is given.
 .PP
 Note that this mode has some potential drawbacks:
 .TP
@@ -300,11 +300,11 @@
 .TP
 \&\fR\&\f(CWDrive\ \fIname\fR\&\f(CW\ is\ mounted\ already\fR\ 
 Trying to mount a drive which appears to be mounted already.  Use the
-\&\fR\&\f(CW--force\fR option if you think this is wrong.
+\&\fR\&\f(CW\-\-force\fR option if you think this is wrong.
 .TP
 \&\fR\&\f(CWDrive\ \fIname\fR\&\f(CW\ is\ not\ mounted\fR\ 
 Trying to unmount a drive which does not appear to be mounted.  Use the
-\&\fR\&\f(CW--force\fR option if you think this is wrong.
+\&\fR\&\f(CW\-\-force\fR option if you think this is wrong.
 .TP
 \&\fR\&\f(CWioctl(...)\ failed\fR\ 
 If this occurs with the \fR\&\f(CWFDGETDRVTYP\fR or \fR\&\f(CWFDGETDRVSTAT\fR,
--- a/doc/fdrawcmd.1
+++ b/doc/fdrawcmd.1
@@ -329,7 +329,7 @@
 means that the driver should not try to autodetect the disk type (it
 might not be formatted), and that it should not reset the FDC. If a
 reset was needed, the command simply fails. If that happens, execute
-\&\fR\&\f(CWfloppycontrol --resetnow 0\fR , and try again.
+\&\fR\&\f(CWfloppycontrol \-\-resetnow 0\fR , and try again.
 .PP
 .SH See Also
 Fdutils' texinfo doc
--- a/doc/floppycontrol.1
+++ b/doc/floppycontrol.1
@@ -72,19 +72,19 @@
 .nf
 .ft 3
 .in +0.3i
-\&\fR\&\f(CWfloppycontrol [\fR\&\f(CW-p] [\fR\&\f(CW--pollstate] [\fR\&\f(CW--printfdstate]
-[\fR\&\f(CW-a \fIoperation-abort-threshold\fR\&\f(CW] [\fR\&\f(CW-c \fIread-track-threshold\fR\&\f(CW]
-[\fR\&\f(CW-r \fIrecalibrate-threshold\fR\&\f(CW] [\fR\&\f(CW-R \fIreset-threshold\fR\&\f(CW]
-[\fR\&\f(CW-e \fIreporting-threshold\fR\&\f(CW] [\fR\&\f(CW-f] [\fR\&\f(CW-x] [\fR\&\f(CW-d \fIdrive\fR\&\f(CW][\fR\&\f(CW-F] [\fR\&\f(CW-T]
-[\fR\&\f(CW-reset \fIcondition\fR\&\f(CW] [\fR\&\f(CW--debug] [\fR\&\f(CW--nodebug] [\fR\&\f(CW--messages]
-[\fR\&\f(CW--nomessages] [\fR\&\f(CW--broken_dcl] [\fR\&\f(CW--working_dcl] [\fR\&\f(CW--inverted_dcl]
-[\fR\&\f(CW--no_inverted_dcl] [\fR\&\f(CW--silent_dcl_clear] [\fR\&\f(CW--noisy_dcl_clear]
-[\fR\&\f(CW-c\fIcmos-type\fR\&\f(CW] [\fR\&\f(CW-hlt \fIhlt\fR\&\f(CW] [\fR\&\f(CW-hut \fIhut\fR\&\f(CW] [\fR\&\f(CW-srt \fIsrt\fR\&\f(CW] [\fR\&\f(CW-o \fIspindown\fR\&\f(CW]
-[\fR\&\f(CW-u \fIspinup\fR\&\f(CW] [\fR\&\f(CW-s \fIselect-delay\fR\&\f(CW] [\fR\&\f(CW-rps \fIrotations-per-second\fR\&\f(CW]
-[\fR\&\f(CW-O \fIspindown-offset\fR\&\f(CW] [\fR\&\f(CW-track \fImax-tracks\fR\&\f(CW] [\fR\&\f(CW-timeout \fIseconds\fR\&\f(CW]
-[\fR\&\f(CW-C \fIcheck-interval\fR\&\f(CW] [\fR\&\f(CW-n \fInative-format\fR\&\f(CW]
-[\fR\&\f(CW-autodetect \fIautodetection-sequence\fR\&\f(CW] [\fR\&\f(CW-P] [\fR\&\f(CW--clrwerror]
-[\fR\&\f(CW--printwerror] [\fR\&\f(CW-h]
+\&\fR\&\f(CWfloppycontrol [\fR\&\f(CW\-p] [\fR\&\f(CW\-\-pollstate] [\fR\&\f(CW\-\-printfdstate]
+[\fR\&\f(CW\-a \fIoperation\-abort\-threshold\fR\&\f(CW] [\fR\&\f(CW\-c \fIread\-track\-threshold\fR\&\f(CW]
+[\fR\&\f(CW\-r \fIrecalibrate\-threshold\fR\&\f(CW] [\fR\&\f(CW\-R \fIreset\-threshold\fR\&\f(CW]
+[\fR\&\f(CW\-e \fIreporting\-threshold\fR\&\f(CW] [\fR\&\f(CW\-f] [\fR\&\f(CW\-x] [\fR\&\f(CW\-d \fIdrive\fR\&\f(CW][\fR\&\f(CW\-F] [\fR\&\f(CW\-T]
+[\fR\&\f(CW\-reset \fIcondition\fR\&\f(CW] [\fR\&\f(CW\-\-debug] [\fR\&\f(CW\-\-nodebug] [\fR\&\f(CW\-\-messages]
+[\fR\&\f(CW\-\-nomessages] [\fR\&\f(CW\-\-broken_dcl] [\fR\&\f(CW\-\-working_dcl] [\fR\&\f(CW\-\-inverted_dcl]
+[\fR\&\f(CW\-\-no_inverted_dcl] [\fR\&\f(CW\-\-silent_dcl_clear] [\fR\&\f(CW\-\-noisy_dcl_clear]
+[\fR\&\f(CW\-c\fIcmos\-type\fR\&\f(CW] [\fR\&\f(CW\-hlt \fIhlt\fR\&\f(CW] [\fR\&\f(CW\-hut \fIhut\fR\&\f(CW] [\fR\&\f(CW\-srt \fIsrt\fR\&\f(CW] [\fR\&\f(CW\-o \fIspindown\fR\&\f(CW]
+[\fR\&\f(CW\-u \fIspinup\fR\&\f(CW] [\fR\&\f(CW\-s \fIselect\-delay\fR\&\f(CW] [\fR\&\f(CW\-rps \fIrotations\-per\-second\fR\&\f(CW]
+[\fR\&\f(CW\-O \fIspindown\-offset\fR\&\f(CW] [\fR\&\f(CW\-track \fImax\-tracks\fR\&\f(CW] [\fR\&\f(CW\-timeout \fIseconds\fR\&\f(CW]
+[\fR\&\f(CW\-C \fIcheck\-interval\fR\&\f(CW] [\fR\&\f(CW\-n \fInative\-format\fR\&\f(CW]
+[\fR\&\f(CW\-autodetect \fIautodetection\-sequence\fR\&\f(CW] [\fR\&\f(CW\-P] [\fR\&\f(CW\-\-clrwerror]
+[\fR\&\f(CW\-\-printwerror] [\fR\&\f(CW\-h]
 .fi
 .in -0.3i
 .ft R
@@ -96,14 +96,14 @@
 .SH General\ Options
 .IP
 .TP
-\&\fR\&\f(CW-h\fR\ 
+\&\fR\&\f(CW\-h\fR\ 
 .TQ
-\&\fR\&\f(CW--help\fR
+\&\fR\&\f(CW\-\-help\fR
 Print a help screen.
 .TP
-\&\fR\&\f(CW-d\ \fIdrive\fR\&\f(CW\ \fR\ 
+\&\fR\&\f(CW\-d\ \fIdrive\fR\&\f(CW\ \fR\ 
 .TQ
-\&\fR\&\f(CW--drive\ \fI\ drive\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-drive\ \fI\ drive\fR\&\f(CW\fR
 Selects the drive to configure. The default is drive 0
 (\fR\&\f(CW\(if/dev/fd0\(is\fR).
 .PP
@@ -113,18 +113,18 @@
 parameter, but perform a one-time action. They are available to anybody
 who has write access to the drive
 .TP
-\&\fR\&\f(CW-f\ \ \fR\ 
+\&\fR\&\f(CW\-f\ \ \fR\ 
 .TQ
-\&\fR\&\f(CW--flush\fR
+\&\fR\&\f(CW\-\-flush\fR
 Flushes (throws away) the dirty data buffers associated with this drive.
 .TP
-\&\fR\&\f(CW-x\ \ \fR\ 
+\&\fR\&\f(CW\-x\ \ \fR\ 
 .TQ
-\&\fR\&\f(CW--eject\fR
+\&\fR\&\f(CW\-\-eject\fR
 Ejects the disk out of the drive (Sparc). The dirty buffers are first
 committed to disk before ejecting it. Fails if the disk is mounted.
 .TP
-\&\fR\&\f(CW--reset\ \fI\ condition\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-\-reset\ \fI\ condition\fR\&\f(CW\fR\ 
 Resets the FDC under
 \&\fIcondition\fR . Condition may be one of the following:
 .RS
@@ -144,9 +144,9 @@
 (see section  fdrawcmd).
 .IP
 .TP
-\&\fR\&\f(CW-F\fR\ 
+\&\fR\&\f(CW\-F\fR\ 
 .TQ
-\&\fR\&\f(CW--formatend\fR
+\&\fR\&\f(CW\-\-formatend\fR
 Issues an end format ioctl. This might be needed after exiting a
 \&\fR\&\f(CWfdformat\fR in an unclean way. \fR\&\f(CWsuperformat\fR is not subject to
 this.
@@ -154,9 +154,9 @@
 .SH Printing\ current\ settings
 .IP
 .TP
-\&\fR\&\f(CW-T\fR\ 
+\&\fR\&\f(CW\-T\fR\ 
 .TQ
-\&\fR\&\f(CW--type\fR
+\&\fR\&\f(CW\-\-type\fR
 Print out the drive name of a floppy device. This is used by the
 \&\fR\&\f(CWMAKEFLOPPIES\fR script. The drive name is a letter (describing the
 drive type) followed by the capacity of the format in bytes. The letter
@@ -168,15 +168,15 @@
 "native format" of the drive is printed, and for the default formats, if
 a generic format has been redefined, its name becomes \fR\&\f(CW(null)\fR.
 .TP
-\&\fR\&\f(CW-p\fR\ 
+\&\fR\&\f(CW\-p\fR\ 
 .TQ
-\&\fR\&\f(CW--print\fR
+\&\fR\&\f(CW\-\-print\fR
 Prints out the configuration of the drive. The names of the various
 fields are the same as the names of the option to set them, see below.
 .TP
-\&\fR\&\f(CW-P\fR\ 
+\&\fR\&\f(CW\-P\fR\ 
 .TQ
-\&\fR\&\f(CW--printstate\fR
+\&\fR\&\f(CW\-\-printstate\fR
 Prints out the cached internal state of the driver. The first line lists
 various attributes about the disk:
 .RS
@@ -207,15 +207,15 @@
 .RS
 .TP
 * \ \ 
--1 means that the driver doesn't know, but the controller does (a seek
+\-1 means that the driver doesn't know, but the controller does (a seek
 command must be issued).
 .TP
 * \ \ 
--2 means that the controller doesn't know either, but is sure that it
+\-2 means that the controller doesn't know either, but is sure that it
 not beyond the 80th track.  The drive needs a recalibration.
 .TP
 * \ \ 
--3 means that the head may be beyond the 80th track.  The drive needs
+\-3 means that the head may be beyond the 80th track.  The drive needs
 two successive recalibrations, because at each recalibration, the
 controller only issues 80 move head commands per recalibration.
 .RE
@@ -253,12 +253,12 @@
 .RE
 .IP
 .TP
-\&\fR\&\f(CW--pollstate\fR\ 
+\&\fR\&\f(CW\-\-pollstate\fR\ 
 Polls the drive and then prints out the internal state of the
-driver.(\fR\&\f(CW--Printstate\fR only prints out the cached information
+driver.(\fR\&\f(CW\-\-Printstate\fR only prints out the cached information
 without actually polling the drive for a disk change.)
 .TP
-\&\fR\&\f(CW--printfdcstate\fR\ 
+\&\fR\&\f(CW\-\-printfdcstate\fR\ 
 Prints out the state of the controller where the target drive is
 attached to.
 .RS
@@ -323,9 +323,9 @@
 .PP
 These settings are only changeable by the super user.
 .TP
-\&\fR\&\f(CW-c\ \fIcmos-type\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-c\ \fIcmos\-type\fR\&\f(CW\fR\ 
 .TQ
-\&\fR\&\f(CW--cmos\ \fI\ cmos-type\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-cmos\ \fI\ cmos\-type\fR\&\f(CW\fR
 Set the virtual CMOS type of the floppy drive. This is useful if
 .RS
 .TP
@@ -351,9 +351,9 @@
 autodetection list and the native format, whereas \fR\&\f(CWfloppycontrol\fR
 does not.
 .TP
-\&\fR\&\f(CW-A\ \ \fIautodetect-seq\fR\&\f(CW\ \fR\ 
+\&\fR\&\f(CW\-A\ \ \fIautodetect\-seq\fR\&\f(CW\ \fR\ 
 .TQ
-\&\fR\&\f(CW--autodetect\ \fI\ autodetect-seq\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-autodetect\ \fI\ autodetect\-seq\fR\&\f(CW\fR
 Set the autodetection sequence (see section  Autodetection) The autodetection
 sequence is a comma-separated list of at most eight format
 descriptors. Each format descriptor is a format number optionally
@@ -396,9 +396,9 @@
 (barring other constraints), as each format that is tried out takes
 400 milliseconds.
 .TP
-\&\fR\&\f(CW-n\ \fInative-format\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-n\ \fInative\-format\fR\&\f(CW\fR\ 
 .TQ
-\&\fR\&\f(CW--native_format\ \fI\ native-format\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-native_format\ \fI\ native\-format\fR\&\f(CW\fR
 Set the native format of this drive. The native format of a drive is the
 highest standard format available for this drive. (Example: For a 5 1/4
 HD drive it is the usual 1200K format.) This is format is used to make
@@ -409,7 +409,7 @@
 .PP
 .SH Configuration\ of\ the\ disk\ change\ line
 .TP
-\&\fR\&\f(CW--broken_dcl\fR\ 
+\&\fR\&\f(CW\-\-broken_dcl\fR\ 
 Assumes that the disk change line of the drive is broken.  If this is
 set, disk changes are assumed to happen whenever the device node is
 first opened. The physical disk change line is ignored.
@@ -441,20 +441,20 @@
 \&\fR\&\f(CWbroken_dcl\fR option on.
 .RE
 .TP
-\&\fR\&\f(CW--working_dcl\fR\ 
+\&\fR\&\f(CW\-\-working_dcl\fR\ 
 Assumes that the disk change line works all right. Switching from
 broken to working may lead to unexpected results after the first disk
 change.
 .TP
-\&\fR\&\f(CW--inverted_dcl\fR\ 
+\&\fR\&\f(CW\-\-inverted_dcl\fR\ 
 Assumes that this disk drive uses an inverted disk change
 line. Apparently this is the case for IBM thinkpads.
 .TP
-\&\fR\&\f(CW--no_inverted_dcl\fR\ 
+\&\fR\&\f(CW\-\-no_inverted_dcl\fR\ 
 Assumes that this drive follows the standard convention for the disk
 change line.
 .TP
-\&\fR\&\f(CW--noisy_dcl_clear\fR\ 
+\&\fR\&\f(CW\-\-noisy_dcl_clear\fR\ 
 Switches off silent disk change line clearing for this drive.
 .PP
 .SH Timing\ Parameters
@@ -463,26 +463,26 @@
 parameters, you need superuser privileges. All times are in "jiffy"
 units (10 milliseconds), unless otherwise specified.
 .TP
-\&\fR\&\f(CW--hlt\ \fI\ hlt\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-\-hlt\ \fI\ hlt\fR\&\f(CW\fR\ 
 Set the head load time (in microseconds) for this floppy drive.  The
 head load time describes how long the floppy controller waits after
 seeking or changing heads before allowing access to a track.
 .TP
-\&\fR\&\f(CW--hut\ \fI\ hut\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-\-hut\ \fI\ hut\fR\&\f(CW\fR\ 
 Set the head unload time (in microseconds) for this floppy drive.  The
 head unload time describes how long the floppy controller waits after an
 access before directing its attention to the other head, or before
 seeking.
 .TP
-\&\fR\&\f(CW--srt\ \fI\ srt\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-\-srt\ \fI\ srt\fR\&\f(CW\fR\ 
 Set the step rate (in microseconds) for this floppy drive.  The step
 rate describes how long the drive head stays on one cylinder when
 seeking.  Setting this value to low (too fast seeks) may make seeks
 fail, because the motor doesn't follow fast enough.
 .TP
-\&\fR\&\f(CW-u\ \fIspinup-time\fR\&\f(CW\ \fR\ 
+\&\fR\&\f(CW\-u\ \fIspinup\-time\fR\&\f(CW\ \fR\ 
 .TQ
-\&\fR\&\f(CW--spinup\ \fI\ spinup-time\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-spinup\ \fI\ spinup\-time\fR\&\f(CW\fR
 Set the spinup time of the floppy drive. In order to do read or write
 to the floppy disk, it must spin. It takes a certain time for the
 motor to reach enough speed to read or write. This parameter describes
@@ -490,17 +490,17 @@
 the spinup time has elapsed. With modern controllers, you may set this time
 to zero, as the controller itself enforces the right delay.
 .TP
-\&\fR\&\f(CW-o\ \fIspindown-time\fR\&\f(CW\ \fR\ 
+\&\fR\&\f(CW\-o\ \fIspindown\-time\fR\&\f(CW\ \fR\ 
 .TQ
-\&\fR\&\f(CW--spindown\ \fI\ spindown-time\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-spindown\ \fI\ spindown\-time\fR\&\f(CW\fR
 Set the spindown time of this floppy drive. The motor is not stopped
 immediately after the operation completes, because there might be more
 operations following. The spindown time is the time the driver waits
 before switching off the motor.
 .TP
-\&\fR\&\f(CW-O\ \fIspindown-offset\fR\&\f(CW\ \fR\ 
+\&\fR\&\f(CW\-O\ \fIspindown\-offset\fR\&\f(CW\ \fR\ 
 .TQ
-\&\fR\&\f(CW--spindown_offset\ \fI\ spindown-offset\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-spindown_offset\ \fI\ spindown\-offset\fR\&\f(CW\fR
 Set the spindown offset of this floppy drive. This parameter is used
 to set the position in which the disk stops. This is useful to
 minimize the next access time. (If the first sector is just near the
@@ -511,16 +511,16 @@
 completes, and using this time to calculate the current position of
 the disk.
 .TP
-\&\fR\&\f(CW-s\ \fIselect-delay\fR\&\f(CW\ \fR\ 
+\&\fR\&\f(CW\-s\ \fIselect\-delay\fR\&\f(CW\ \fR\ 
 .TQ
-\&\fR\&\f(CW--select_delay\ \fI\ select-delay\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-select_delay\ \fI\ select\-delay\fR\&\f(CW\fR
 Set the \fIselect delay\fR of this floppy drive. This is the delay that
 the driver waits after selecting the drive and issuing the first command
 to it. For modern controllers/drives, you may set this to zero.
 .TP
-\&\fR\&\f(CW-C\ \fIcheck-interval\fR\&\f(CW\ \fR\ 
+\&\fR\&\f(CW\-C\ \fIcheck\-interval\fR\&\f(CW\ \fR\ 
 .TQ
-\&\fR\&\f(CW--checkfreq\ \fI\ check-interval\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-checkfreq\ \fI\ check\-interval\fR\&\f(CW\fR
 Set the maximal disk change check interval.  The disk change line is
 checked whenever a read or write to the device is issued, and it has not
 been checked for more than \fIinterval\fR jiffies.
@@ -531,20 +531,20 @@
 on and off.
 .IP
 .TP
-\&\fR\&\f(CW--debug\fR\ 
+\&\fR\&\f(CW\-\-debug\fR\ 
 Switch debugging output on. The debugging information includes timing
 information. This option might be useful to fine-tune the timing
 options for your local setups. (But for most normal purposes, the
 default values are good enough.)
 .TP
-\&\fR\&\f(CW--nodebug\fR\ 
+\&\fR\&\f(CW\-\-nodebug\fR\ 
 Switch debugging output off.
 .TP
-\&\fR\&\f(CW--messages\fR\ 
+\&\fR\&\f(CW\-\-messages\fR\ 
 Print informational messages after autodetection, geometry parameter
 clearing and dma over/underruns.
 .TP
-\&\fR\&\f(CW--nomessages\fR\ 
+\&\fR\&\f(CW\-\-nomessages\fR\ 
 Don't print informational messages after these events.
 .PP
 .SH Error\ Handling\ Options
@@ -557,38 +557,38 @@
 are performed at the next retry. The counter is reset when the read or
 write finally terminates, whether successfully or not.
 .TP
-\&\fR\&\f(CW-a\ \fIoperation-abort-threshold\fR\&\f(CW\ \fR\ 
+\&\fR\&\f(CW\-a\ \fIoperation\-abort\-threshold\fR\&\f(CW\ \fR\ 
 .TQ
-\&\fR\&\f(CW--abort\ \fI\ operation-abort-threshold\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-abort\ \fI\ operation\-abort\-threshold\fR\&\f(CW\fR
 Tell the floppy driver to stop trying to read/write a sector after
 \&\fIoperation-abort-threshold\fR
 retries, and signal the I/O error to the user.
 .TP
-\&\fR\&\f(CW-t\ \fIread-track-threshold\fR\&\f(CW\ \fR\ 
+\&\fR\&\f(CW\-t\ \fIread\-track\-threshold\fR\&\f(CW\ \fR\ 
 .TQ
-\&\fR\&\f(CW--readtrack\ \fI\ read-track-threshold\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-readtrack\ \fI\ read\-track\-threshold\fR\&\f(CW\fR
 Tell the floppy driver to switch from track-reading mode to
 sector-at-a-time-mode after
 \&\fIread-track-threshold\fR
 retries.
 .TP
-\&\fR\&\f(CW-r\ \fIrecalibrate-threshold\fR\&\f(CW\ \fR\ 
+\&\fR\&\f(CW\-r\ \fIrecalibrate\-threshold\fR\&\f(CW\ \fR\ 
 .TQ
-\&\fR\&\f(CW--recalibrate\ \fI\ recalibrate-threshold\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-recalibrate\ \fI\ recalibrate\-threshold\fR\&\f(CW\fR
 Tell the floppy driver to recalibrate the drive after
 \&\fIrecalibrate-threshold\fR retries.
 .TP
-\&\fR\&\f(CW-R\ \fIreset-threshold\fR\&\f(CW\ \fR\ 
+\&\fR\&\f(CW\-R\ \fIreset\-threshold\fR\&\f(CW\ \fR\ 
 .TQ
-\&\fR\&\f(CW--reset\ \fI\ reset-threshold\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-reset\ \fI\ reset\-threshold\fR\&\f(CW\fR
 Tell the floppy driver to reset the controller after
 \&\fIreset-threshold\fR retries. After a controller reset, the floppy
 driver also recalibrates all drives connected to that controller.
 .IP
 .TP
-\&\fR\&\f(CW-e\ \fIerror-report-threshold\fR\&\f(CW\ \fR\ 
+\&\fR\&\f(CW\-e\ \fIerror\-report\-threshold\fR\&\f(CW\ \fR\ 
 .TQ
-\&\fR\&\f(CW--reporting\ \fI\ error-report-threshold\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-reporting\ \fI\ error\-report\-threshold\fR\&\f(CW\fR
 Tell the floppy driver to start printing error messages to the console
 after \fIerror-report-threshold\fR retries.
 .PP
@@ -603,10 +603,10 @@
 which wants to make sure that the data has been written successfully.
 .IP
 .TP
-\&\fR\&\f(CW--clrwerror\fR\ 
+\&\fR\&\f(CW\-\-clrwerror\fR\ 
 Clears the write error structure.
 .TP
-\&\fR\&\f(CW--printwerror\fR\ 
+\&\fR\&\f(CW\-\-printwerror\fR\ 
 Prints the contents of the write error structure:
 .RS
 .TP
@@ -639,7 +639,7 @@
 in any other category.  They are available only to the superuser:
 .IP
 .TP
-\&\fR\&\f(CW--tracks\ \fI\ max-tracks\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-\-tracks\ \fI\ max\-tracks\fR\&\f(CW\fR\ 
 Set the maximal numbers of physical tracks that this drive may
 handle. If you have a drive which is only able to handle 80 tracks
 (making strange noises when you try to format or read a disk with more
@@ -651,9 +651,9 @@
 drives you don't need to worry anyways. See section More cylinders, for
 details.
 .TP
-\&\fR\&\f(CW-i\ \fIsector-interleave\fR\&\f(CW\ \fR\ 
+\&\fR\&\f(CW\-i\ \fIsector\-interleave\fR\&\f(CW\ \fR\ 
 .TQ
-\&\fR\&\f(CW--interleave\ \fIsector-interleave\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-interleave\ \fIsector\-interleave\fR\&\f(CW\fR
 Set the number of sectors beyond which sector interleaving will be used.
 This option will only be used by the \fR\&\f(CWFDFMTTRK\fR ioctl.  The
 \&\fR\&\f(CWfdformat\fR command, which is now considered obsolete, uses
--- a/doc/diskseekd.1
+++ b/doc/diskseekd.1
@@ -86,23 +86,23 @@
 .nf
 .ft 3
 .in +0.3i
-\&\fR\&\f(CWdiskseekd [\fR\&\f(CW-d \fIdrive\fR\&\f(CW] [\fR\&\f(CW-i \fIinterval\fR\&\f(CW] [\fR\&\f(CW-p \fIpidfile\fR\&\f(CW]
+\&\fR\&\f(CWdiskseekd [\fR\&\f(CW\-d \fIdrive\fR\&\f(CW] [\fR\&\f(CW\-i \fIinterval\fR\&\f(CW] [\fR\&\f(CW\-p \fIpidfile\fR\&\f(CW]
 .fi
 .in -0.3i
 .ft R
  
 \&\fR
 .TP
-\&\fR\&\f(CW-d\ \fIdrive\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-d\ \fIdrive\fR\&\f(CW\fR\ 
 Selects the drive to seek.  By default, drive 0 (\fR\&\f(CW\(if/dev/fd0\(is\fR) is sought.
 .TP
-\&\fR\&\f(CW-i\ \fIinterval\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-i\ \fIinterval\fR\&\f(CW\fR\ 
 Selects the cleaning interval, in seconds.  If the interval is 0, a
 single seek is done. This is useful when calling diskseek from a
 crontab.  The default is 1000 seconds (about 16 minutes) for
 \&\fR\&\f(CWdiskseekd\fR and 0 for \fR\&\f(CWdiskseek\fR.
 .TP
-\&\fR\&\f(CW-p\ \fIpidfile\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-p\ \fIpidfile\fR\&\f(CW\fR\ 
 Stores the process id of the diskseekd daemon into \fIpidfile\fR instead
 of the default \fR\&\f(CW\(if/var/run/diskseekd.pid\(is\fR.
 .PP
--- a/doc/floppymeter.1
+++ b/doc/floppymeter.1
@@ -72,8 +72,8 @@
 .nf
 .ft 3
 .in +0.3i
-\&\fR\&\f(CWfloppymeter [\fR\&\f(CW-f] [\fR\&\f(CW-w \fIwarmup-delay\fR\&\f(CW] [\fR\&\f(CW-W \fIwindow\fR\&\f(CW]
-[\fR\&\f(CW-c \fIcycles\fR\&\f(CW] [\fR\&\f(CW-h] \fIdrive\fR\&\f(CW
+\&\fR\&\f(CWfloppymeter [\fR\&\f(CW\-f] [\fR\&\f(CW\-w \fIwarmup\-delay\fR\&\f(CW] [\fR\&\f(CW\-W \fIwindow\fR\&\f(CW]
+[\fR\&\f(CW\-c \fIcycles\fR\&\f(CW] [\fR\&\f(CW\-h] \fIdrive\fR\&\f(CW
 .fi
 .in -0.3i
 .ft R
@@ -84,7 +84,7 @@
 floppy drive and floppy controller, such as the rotation speed of the
 drive, the data transfer rate of the controller, and the resulting raw
 capacity of a disk track.  To use this program, insert a disposable
-floppy in the drive, and type \fR\&\f(CWfloppymeter --\fR\fIdensity\fR, where
+floppy in the drive, and type \fR\&\f(CWfloppymeter \-\-\fR\fIdensity\fR, where
 density describes the density of the disk used for the test. (Should be
 one of \fR\&\f(CWdd, hd\fR or \fR\&\f(CWed\fR). \fBCAUTION: the data on the
 disk will be erased\fR.  This program should be used to verify whether the
@@ -125,31 +125,31 @@
 If on the other hand, the raw capacity of the drive is too big, you may
 get problems when writing to a disk formatted by this drive on another
 drive with a smaller raw capacity. In order to avoid this, increase
-superformats gap parameter (\fR\&\f(CW-G\fR).
+superformats gap parameter (\fR\&\f(CW\-G\fR).
 .IP
 .TP
-\&\fR\&\f(CW-h\fR\ 
+\&\fR\&\f(CW\-h\fR\ 
 Prints a short help
 .TP
-\&\fR\&\f(CW--dd\fR\ 
+\&\fR\&\f(CW\-\-dd\fR\ 
 Tells the program that we use a Double Density disk.
 .TP
-\&\fR\&\f(CW--hd\fR\ 
+\&\fR\&\f(CW\-\-hd\fR\ 
 Tells the program that we use a High Density disk.
 .TP
-\&\fR\&\f(CW--ed\fR\ 
+\&\fR\&\f(CW\-\-ed\fR\ 
 Tells the program that we use an Extra Density disk.
 .TP
-\&\fR\&\f(CW-f\fR\ 
+\&\fR\&\f(CW\-f\fR\ 
 Runs the measurement non interactively.  With this option, the program
 doesn't ask for confirmation, and doesn't display the continuously
 updated values during the rotation speed measurement.
 .TP
-\&\fR\&\f(CW-W\ \fIWindow\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-W\ \fIWindow\fR\&\f(CW\fR\ 
 This value describes how many rotations are used for the computation of
 the sliding average. Default is 30.
 .TP
-\&\fR\&\f(CW-c\ \fIcycles\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-c\ \fIcycles\fR\&\f(CW\fR\ 
 Describes the number of rotations clocked during the rotations speed
 determination test. Default is 1000.
 .PP
--- a/doc/makefloppies.1
+++ b/doc/makefloppies.1
@@ -72,7 +72,7 @@
 .nf
 .ft 3
 .in +0.3i
-\&\fR\&\f(CWMAKEFLOPPIES [\fR\&\f(CW-tlvng] [\fIdrives\fR\&\f(CW]
+\&\fR\&\f(CWMAKEFLOPPIES [\fR\&\f(CW\-tlvng] [\fIdrives\fR\&\f(CW]
 .fi
 .in -0.3i
 .ft R
@@ -102,24 +102,24 @@
 .SH Options
 .IP
 .TP
-\&\fR\&\f(CW-t\fR\ 
+\&\fR\&\f(CW\-t\fR\ 
 Use the old naming convention for 3 1/2 devices (e.g. \fR\&\f(CW\(iffd0H720\(is\fR
 instead of \fR\&\f(CW\(iffd0u720\(is\fR).
 .TP
-\&\fR\&\f(CW-m\fR\ 
+\&\fR\&\f(CW\-m\fR\ 
 Base the name for the created devices on the type of the media
 (e.g. \fR\&\f(CW\(iffd0h720\(is\fR instead of \fR\&\f(CW\(iffd0u720\(is\fR).
 .TP
-\&\fR\&\f(CW-l\fR\ 
+\&\fR\&\f(CW\-l\fR\ 
 Local. Creates device nodes in the local directory, not /dev
 .TP
-\&\fR\&\f(CW-v\fR\ 
+\&\fR\&\f(CW\-v\fR\ 
 Verbose
 .TP
-\&\fR\&\f(CW-n\fR\ 
+\&\fR\&\f(CW\-n\fR\ 
 Dry run. (just report what would be done, do not do anything)
 .TP
-\&\fR\&\f(CW-g\fR\ 
+\&\fR\&\f(CW\-g\fR\ 
 Group.  Allow read/write access to floppy devices only for group
 \&\fR\&\f(CW\(iffloppy\(is\fR
 .PP
--- a/doc/setfdprm.1
+++ b/doc/setfdprm.1
@@ -72,9 +72,9 @@
 .nf
 .ft 3
 .in +0.3i
-\&\fR\&\f(CWsetfdprm [\fR\&\f(CW-p] \fIdevice\fR\&\f(CW \fImedia-description\fR\&\f(CW
+\&\fR\&\f(CWsetfdprm [\fR\&\f(CW\-p] \fIdevice\fR\&\f(CW \fImedia\-description\fR\&\f(CW
 \&\&
-\&\fR\&\f(CWsetfdprm [\fR\&\f(CW-c | \fR\&\f(CW-y | \fR\&\f(CW-n] \fIdevice\fR\&\f(CW
+\&\fR\&\f(CWsetfdprm [\fR\&\f(CW\-c | \fR\&\f(CW\-y | \fR\&\f(CW\-n] \fIdevice\fR\&\f(CW
 \&\&
 .fi
 .in -0.3i
@@ -101,20 +101,20 @@
 .SH Options
 .IP
 .TP
-\&\fR\&\f(CW-p\ \fIdevice\ name\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-p\ \fIdevice\ name\fR\&\f(CW\fR\ 
 Permanently loads a new parameter set for the specified auto-configuring
 floppy device for the configuration with \fIname\fR in
 \&\fR\&\f(CW\(if/etc/fdprm\(is\fR. Alternatively, the parameters can be given directly
 from the command line.
 .TP
-\&\fR\&\f(CW-c\ \fIdevice\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-c\ \fIdevice\fR\&\f(CW\fR\ 
 Clears the parameter set of the specified auto-configuring floppy device.
 .TP
-\&\fR\&\f(CW-y\ \fIdevice\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-y\ \fIdevice\fR\&\f(CW\fR\ 
 Enables format detection messages for the specified auto-configuring floppy
 device.
 .TP
-\&\fR\&\f(CW-n\ \fIdevice\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-n\ \fIdevice\fR\&\f(CW\fR\ 
 Disables format detection messages for the specified auto-configuring
 floppy device.
 .PP
--- a/doc/superformat.1
+++ b/doc/superformat.1
@@ -72,12 +72,12 @@
 .nf
 .ft 3
 .in +0.3i
-\&\fR\&\f(CWsuperformat [\fR\&\f(CW-D \fIdos-drive\fR\&\f(CW] [\fR\&\f(CW-v \fIverbosity-level\fR\&\f(CW] [\fR\&\f(CW-b \fIbegin-track\fR\&\f(CW]
-[\fR\&\f(CW-e \fIend-track\fR\&\f(CW] [\fR\&\f(CW--superverify] [\fR\&\f(CW--dosverify]
-[\fR\&\f(CW--noverify] [\fR\&\f(CW--verify_later] [\fR\&\f(CW--first-sector-number \fIn\fR\&\f(CW] [\fR\&\f(CW--zero-based]
-[\fR\&\f(CW-G \fIformat-gap\fR\&\f(CW] [\fR\&\f(CW-F \fIfinal-gap\fR\&\f(CW] [\fR\&\f(CW-i \fIinterleave\fR\&\f(CW] [\fR\&\f(CW-c \fIchunksize\fR\&\f(CW]
-[\fR\&\f(CW-g \fIgap\fR\&\f(CW] [\fR\&\f(CW--absolute-skew \fIabsolute-skew\fR\&\f(CW] [\fR\&\f(CW--head-skew \fIhead-skew\fR\&\f(CW]
-[\fR\&\f(CW--track-skew \fItrack-skew\fR\&\f(CW] [\fR\&\f(CW--biggest-last] \fIdrive\fR\&\f(CW [\fImedia-description\fR\&\f(CW]
+\&\fR\&\f(CWsuperformat [\fR\&\f(CW\-D \fIdos\-drive\fR\&\f(CW] [\fR\&\f(CW\-v \fIverbosity\-level\fR\&\f(CW] [\fR\&\f(CW\-b \fIbegin\-track\fR\&\f(CW]
+[\fR\&\f(CW\-e \fIend\-track\fR\&\f(CW] [\fR\&\f(CW\-\-superverify] [\fR\&\f(CW\-\-dosverify]
+[\fR\&\f(CW\-\-noverify] [\fR\&\f(CW\-\-verify_later] [\fR\&\f(CW\-\-first\-sector\-number \fIn\fR\&\f(CW] [\fR\&\f(CW\-\-zero\-based]
+[\fR\&\f(CW\-G \fIformat\-gap\fR\&\f(CW] [\fR\&\f(CW\-F \fIfinal\-gap\fR\&\f(CW] [\fR\&\f(CW\-i \fIinterleave\fR\&\f(CW] [\fR\&\f(CW\-c \fIchunksize\fR\&\f(CW]
+[\fR\&\f(CW\-g \fIgap\fR\&\f(CW] [\fR\&\f(CW\-\-absolute\-skew \fIabsolute\-skew\fR\&\f(CW] [\fR\&\f(CW\-\-head\-skew \fIhead\-skew\fR\&\f(CW]
+[\fR\&\f(CW\-\-track\-skew \fItrack\-skew\fR\&\f(CW] [\fR\&\f(CW\-\-biggest\-last] \fIdrive\fR\&\f(CW [\fImedia\-description\fR\&\f(CW]
 .fi
 .in -0.3i
 .ft R
@@ -119,51 +119,51 @@
 .SH Common\ Options
 Many options have a long and a short form.
 .TP
-\&\fR\&\f(CW-h\fR\ 
+\&\fR\&\f(CW\-h\fR\ 
 .TQ
-\&\fR\&\f(CW--help\fR
+\&\fR\&\f(CW\-\-help\fR
 Print the help.
 .TP
-\&\fR\&\f(CW-D\ \fIdrive\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-D\ \fIdrive\fR\&\f(CW\fR\ 
 .TQ
-\&\fR\&\f(CW--dosdrive\ \fIdos-drive\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-dosdrive\ \fIdos\-drive\fR\&\f(CW\fR
 Selects DOS drive letter for \fR\&\f(CWmformat\fR (for example \fR\&\f(CWa:\fR or
 \&\fR\&\f(CWb:\fR).  The colon may be omitted.  The default is derived from the
 minor device number.  If the drive letter cannot be guessed, and is not
 given on the command line, \fR\&\f(CWmformat\fR is skipped.
 .TP
-\&\fR\&\f(CW-v\ \fIverbosity-level\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-v\ \fIverbosity\-level\fR\&\f(CW\fR\ 
 .TQ
-\&\fR\&\f(CW--verbosity\ \fIverbosity-level\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-verbosity\ \fIverbosity\-level\fR\&\f(CW\fR
 Sets the verbosity level. 1 prints a dot for each formatted track. 2
-prints a changing sign for each formatted track (- for formatting the
+prints a changing sign for each formatted track (\- for formatting the
 first head, = for formatting the second head, x for verifying the
 first head, and + for verifying the second head). 3 prints a complete
 line listing head and track. 6 and 9 print debugging information.
 .TP
-\&\fR\&\f(CW--superverify\fR\ 
+\&\fR\&\f(CW\-\-superverify\fR\ 
 Verifies the disk by first reading the track, than writing a pattern of
 U's, and then reading it again.  This is useful as some errors only show
 up after the disk has once been written.  However, this is also slower.
 .TP
-\&\fR\&\f(CW-B\fR\ 
+\&\fR\&\f(CW\-B\fR\ 
 .TQ
-\&\fR\&\f(CW--dosverify\fR
+\&\fR\&\f(CW\-\-dosverify\fR
 Verifies the disk using the \fR\&\f(CWmbadblocks\fR program.
 \&\fR\&\f(CWmbadblocks\fR marks the bad sectors as bad in the FAT.  The
 advantage of this is that disks which are only partially bad can still
 be used for MS-DOS filesystems.
 .TP
-\&\fR\&\f(CW-V\fR\ 
+\&\fR\&\f(CW\-V\fR\ 
 .TQ
-\&\fR\&\f(CW--verify_later\fR
+\&\fR\&\f(CW\-\-verify_later\fR
 Verifies the whole disk at the end of the formatting process instead
 of at each track. Verifying the disk at each track has the advantage
 of detecting errors early on.
 .TP
-\&\fR\&\f(CW-f\fR\ 
+\&\fR\&\f(CW\-f\fR\ 
 .TQ
-\&\fR\&\f(CW--noverify\fR
+\&\fR\&\f(CW\-\-noverify\fR
 Skips the verification altogether.
 .PP
 .SH Advanced\ Options
@@ -173,70 +173,70 @@
 or superformat itself has bugs.
 .IP
 .TP
-\&\fR\&\f(CW-b\ \fIbegin-track\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-b\ \fIbegin\-track\fR\&\f(CW\fR\ 
 .TQ
-\&\fR\&\f(CW--begin_track\ \ \fIbegin-track\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-begin_track\ \ \fIbegin\-track\fR\&\f(CW\fR
 Describes the track where to begin formatting.  This is useful if the
 previous formatting failed halfway through.  The default is 0.
 .TP
-\&\fR\&\f(CW-e\ \fIend-track\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-e\ \fIend\-track\fR\&\f(CW\fR\ 
 .TQ
-\&\fR\&\f(CW--end_track\ \fIend-track\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-end_track\ \fIend\-track\fR\&\f(CW\fR
 Describes where to stop formatting. \fIend_track\fR is the last track to
 be formatted plus one. This is mainly useful for testing purposes. By
 default, this is the same as the total number of tracks.  When the
 formatting stops, the final skew is displayed (to be used as absolute
 skew when you'll continue).
 .TP
-\&\fR\&\f(CW-S\ \fIsizecode\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-S\ \fIsizecode\fR\&\f(CW\fR\ 
 .TQ
-\&\fR\&\f(CW--sizecode\ \fIsizecode\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-sizecode\ \fIsizecode\fR\&\f(CW\fR
 Set the sector size to be used. The sector size is 128 * (2 ^
 \&\fIsizecode\fR).  Sector sizes below 512 bytes are not supported, thus
 sizecode must be at least 2. By default 512 is assumed, unless you ask
 for more sectors than would fit with 512 bytes.
 .TP
-\&\fR\&\f(CW--stretch\ \fIstretch\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-\-stretch\ \fIstretch\fR\&\f(CW\fR\ 
 Set the stretch factor. The stretch factor describes how many physical
 tracks to skip to get to the next logical track (2 ^ \fIstretch\fR).  On
 double density 5 1/4 disks, the tracks are further apart from each
 other.
 .TP
-\&\fR\&\f(CW-G\ \fIfmt-gap\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-G\ \fIfmt\-gap\fR\&\f(CW\fR\ 
 .TQ
-\&\fR\&\f(CW--format_gap\ \fIfmt-gap\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-format_gap\ \fIfmt\-gap\fR\&\f(CW\fR
 Set the formatting gap. The formatting gap tells how far the sectors
 are away from each other. By default, this is chosen so as to evenly
 distribute the sectors along the track.
 .TP
-\&\fR\&\f(CW-F\ \fIfinal-gap\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-F\ \fIfinal\-gap\fR\&\f(CW\fR\ 
 .TQ
-\&\fR\&\f(CW--final_gap\ \fIfinal-gap\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-final_gap\ \fIfinal\-gap\fR\&\f(CW\fR
 Set the formatting gap to be used after the last sector.
 .TP
-\&\fR\&\f(CW-i\ \fIinterleave\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-i\ \fIinterleave\fR\&\f(CW\fR\ 
 .TQ
-\&\fR\&\f(CW--interleave\ \fIinterleave\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-interleave\ \fIinterleave\fR\&\f(CW\fR
 Set the sector interleave factor.
 .TP
-\&\fR\&\f(CW-c\ \fIchunksize\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-c\ \fIchunksize\fR\&\f(CW\fR\ 
 .TQ
-\&\fR\&\f(CW--chunksize\ \fIchunksize\fR\&\f(CW\fR
+\&\fR\&\f(CW\-\-chunksize\ \fIchunksize\fR\&\f(CW\fR
 Set the size of the chunks. The chunks are small auxiliary sectors
 used during formatting. They are used to handle heterogeneous sector
 sizes (i.e. not all sectors have the same size) and negative
 formatting gaps.
 .TP
-\&\fR\&\f(CW--biggest-last\fR\ 
+\&\fR\&\f(CW\-\-biggest\-last\fR\ 
 For MSS formats, make sure that the biggest sector is last on the track.
 This makes the format more reliable on drives which are out of spec.
 .TP
-\&\fR\&\f(CW--first-sector-number\ \fIn\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-\-first\-sector\-number\ \fIn\fR\&\f(CW\fR\ 
 Formats the disk with sector numbers starting at \fIn\fR, rather than
 1. Certain CP/M boxes or Music synthesizers use this format.
 .TP
-\&\fR\&\f(CW--zero-based\fR\ 
-Shorthand for \fR\&\f(CW--first-sector-number 0\fR
+\&\fR\&\f(CW\-\-zero\-based\fR\ 
+Shorthand for \fR\&\f(CW\-\-first\-sector\-number 0\fR
 .PP
 .SH Sector\ skewing\ options
 .PP
@@ -250,17 +250,17 @@
 and track. The amount of this change depends on how fast the disk
 spins, and on how much time is needed to change the head or the track.
 .TP
-\&\fR\&\f(CW--absolute_skew\ \fIabsolute-skew\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-\-absolute_skew\ \fIabsolute\-skew\fR\&\f(CW\fR\ 
 .IP
 Set the absolute skew. This skew value is used for the first formatted
 track.  It is expressed in raw bytes.
 .TP
-\&\fR\&\f(CW--head_skew\ \fIhead-skew\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-\-head_skew\ \fIhead\-skew\fR\&\f(CW\fR\ 
 .IP
 Set the head skew. This is the skew added for passing from head 0 to
 head 1.  It is expressed in raw bytes.
 .TP
-\&\fR\&\f(CW--track_skew\ \fItrack-skew\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-\-track_skew\ \fItrack\-skew\fR\&\f(CW\fR\ 
 .IP
 Set the track skew. This is the skew added for seeking to the next
 track.  It is expressed in raw bytes.
--- a/doc/xdfcopy.1
+++ b/doc/xdfcopy.1
@@ -72,8 +72,8 @@
 .nf
 .ft 3
 .in +0.3i
-\&\fR\&\f(CWxdfcopy [\fR\&\f(CW-\fIformat-id\fR\&\f(CW] [\fR\&\f(CW-d] [\fR\&\f(CW-n] [\fR\&\f(CW-h \fIhead-skew\fR\&\f(CW] [\fR\&\f(CW-t \fIcylinder-skew\fR\&\f(CW] [\fR\&\f(CW-T
-\&\fIend-cylinder\fR\&\f(CW] [\fIsource\fR\&\f(CW] \fItarget\fR\&\f(CW
+\&\fR\&\f(CWxdfcopy [\fR\&\f(CW\-\fIformat\-id\fR\&\f(CW] [\fR\&\f(CW\-d] [\fR\&\f(CW\-n] [\fR\&\f(CW\-h \fIhead\-skew\fR\&\f(CW] [\fR\&\f(CW\-t \fIcylinder\-skew\fR\&\f(CW] [\fR\&\f(CW\-T
+\&\fIend\-cylinder\fR\&\f(CW] [\fIsource\fR\&\f(CW] \fItarget\fR\&\f(CW
 .fi
 .in -0.3i
 .ft R
@@ -91,7 +91,7 @@
 .PP
 If both source and target are given, xdfcopy copies the disk image from
 file to floppy disk or vice-versa. When copying to a floppy disk, the
-disk is first formatted, unless the \fR\&\f(CW-n\fR option is given.
+disk is first formatted, unless the \fR\&\f(CW\-n\fR option is given.
 .PP
 If no source is given, the target is only formatted. In this case, the
 target must be a floppy drive.
@@ -120,7 +120,7 @@
 .PP
 .SH Misc\ options
 .TP
-\&\fR\&\f(CW-D\ \fIdosdrive\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-D\ \fIdosdrive\fR\&\f(CW\fR\ 
 Describes the DOS drive letter for mformat. If this option is given, an
 MS-DOS filesystem is automatically installed on the disk after the
 low-level format is complete. In order for this to work, the drive has
@@ -152,27 +152,27 @@
  
 \&\fR
 .TP
-\&\fR\&\f(CW-n\fR\ 
+\&\fR\&\f(CW\-n\fR\ 
 Don't format the disk before copying the disk image to the disk.
 .PP
 .SH Options\ for\ power\ users
 .IP
 .TP
-\&\fR\&\f(CW-t\ \fIcylinder\ skew\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-t\ \fIcylinder\ skew\fR\&\f(CW\fR\ 
 Uses a different track skew than the default (14). For more details on
-skews, see section  superformat. In this version of xdfcopy, the \fR\&\f(CW-t\fR
+skews, see section  superformat. In this version of xdfcopy, the \fR\&\f(CW\-t\fR
 parameter is ignored.
 .TP
-\&\fR\&\f(CW-h\ \fIhead\ skew\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-h\ \fIhead\ skew\fR\&\f(CW\fR\ 
 Uses a different head skew than the default (0) In this version, this
 parameter is ignored
 .TP
-\&\fR\&\f(CW-d\fR\ 
+\&\fR\&\f(CW\-d\fR\ 
 Debugging. For each read or write operation, the time it took to
 complete the operation is printed (in milliseconds). This can be used
 to optimize the skews.
 .TP
-\&\fR\&\f(CW-T\ \fIend-cylinders\fR\&\f(CW\fR\ 
+\&\fR\&\f(CW\-T\ \fIend\-cylinders\fR\&\f(CW\fR\ 
 Tells how many cylinders to format.  With the XXDF formats, it is
 actually possible to format up to 83 cylinders, yielding a format of
 up to 1992KB on a 3 1/2 high density disk.
